pkl-core/src/test/kotlin/org/pkl/core/LanguageSnippetTestsEngine.kt

```diff
@@ -79,6 +79,9 @@ abstract class AbstractLanguageSnippetTestsEngine : InputOutputTestEngine() {
   // can't think of a better solution right now
   protected fun String.stripVersionCheckErrorMessage() =
     replace("Pkl version is ${Release.current().version()}", "Pkl version is xxx")
+  
+  protected fun String.stripStdlibLocationSha(): String =
+    replace(Regex("https://github.com/apple/pkl/blob/[0-9a-fA-F]+/stdlib/"), "https://github.com/apple/pkl/blob/stdlib/")
```

Comment:
It is generally wise to leave some marker that something was removed (compare, for example, `xxx` in `stripVersionCheckErrorMessage`).
```suggestion
    replace(Regex("https://github.com/apple/pkl/blob/[0-9a-fA-F]+/stdlib/"), "https://github.com/apple/pkl/blob/__hash__/stdlib/")
```

Reply:
Done